lib/v2/screens/profile_screens/citizenship/citizenship_screen.dart

```diff
@@ -0,0 +1,195 @@
+  AnimationController _controller;
+  Animation<double> _timeLineAnimation,
+      _reputationAnimation,
+      _visitorsAnimation,
+      _ageAnimation,
+      _seedsAnimation,
+      _transactionsAnimation,
+      _friendsAnimation;
+  int _timeLine = 0, _reputation = 0, _visitors = 0, _age = 0, _seeds = 0, _transactions = 0, _friends = 0;
+
+  @override
+  void initState() {
+    _controller = AnimationController(duration: const Duration(seconds: 1), vsync: this);
+    _timeLineAnimation = Tween<double>(begin: 0, end: 85).animate(_controller)
+      ..addListener(() {
+        setState(() => _timeLine = _timeLineAnimation.value.toInt());
+      });
+    _reputationAnimation = Tween<double>(begin: 0, end: 50).animate(_controller)
+      ..addListener(() {
+        setState(() => _reputation = _reputationAnimation.value.toInt());
+      });
+    _visitorsAnimation = Tween<double>(begin: 0, end: 1).animate(_controller)
+      ..addListener(() {
+        setState(() => _visitors = _visitorsAnimation.value.toInt());
+      });
+    _ageAnimation = Tween<double>(begin: 0, end: 59).animate(_controller)
+      ..addListener(() {
+        setState(() => _age = _ageAnimation.value.toInt());
+      });
+    _seedsAnimation = Tween<double>(begin: 0, end: 200).animate(_controller)
+      ..addListener(() {
+        setState(() => _seeds = _seedsAnimation.value.toInt());
+      });
+    _transactionsAnimation = Tween<double>(begin: 0, end: 3).animate(_controller)
+      ..addListener(() {
+        setState(() => _transactions = _transactionsAnimation.value.toInt());
+      });
+    _friendsAnimation = Tween<double>(begin: 0, end: 1).animate(_controller)
+      ..addListener(() {
+        setState(() => _friends = _friendsAnimation.value.toInt());
+      });
+    _controller.forward();
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _controller.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    const ProfileAvatar(
+                      size: 100,
+                      image: '',
+                      nickname: 'raul',
+                      account: 'raul',
+                    ),
+                    const SizedBox(height: 8.0),
+                    Text(
+                      'Raul',
+                      style: Theme.of(context).textTheme.headline6,
+                    ),
+                    const SizedBox(height: 8.0),
+                    Text(
+                      'Resident',
+                      style: Theme.of(context).textTheme.headline7LowEmphasis,
```

Comment:
designs dont match to this text style. 
I see he used Button style but i tried that and its missing the 50% opacity. 

Please tell Jason we dont have button with 50% opacity and that he needs to fix that in the designs.